index.js

```diff
@@ -166,6 +217,17 @@ const line = []
 
 // const newLine = []
 // const takeATicketNumberFunc = ticketNumberGeneratorFunc()
+
+// function ticketNumberGeneratorFunc(){
+
+//       MY MIND IS BLOWN HERE!
+      // MY MIND IS BLOWN HERE!
+      // MY MIND IS BLOWN HERE!
+      // MY MIND IS BLOWN HERE!
+      // MY MIND IS BLOWN HERE!
+
+// }
```

Comment:
his one's a tricky bonus one! Check this Scrimba out [https://scrimba.com/scrim/cLgEmZUZ ](https://scrimba.com/scrim/cLgEmZUZ ) and try again. Don't worry if you don't get it right now; It's just a bonus and it involves advanced, tricky concepts and this is only your first day with JavaScript!